README.md

```diff
@@ -120,49 +166,142 @@
 |foundDate|Date|N|-|팀 창단일|
 |UID|String|Y|-|대표자|
 
-#### 2.4 팀 삭제
+### 출력 결과
+
+|필드|타입|설명|
+|---|---|---|
+|status|int|200 : 수정 성공, 500 : 수정 실패|
+|myteamPageUrl|String|마이팀페이지 URL|
+
+## 팀 삭제
+
+### API 기본정보
+
+|메서드|요청URL|출력포맷|설명|
+|---|---|---|---|
+|DELETE|/teams/delete/{TID}|json|팀 삭제|
+
+### 요청 변수
 
 |요청변수명|타입|필수 여부|기본값|설명|
 |---|---|---|---|---|
 |TID|String|Y|-|팀 식별자|
 
+### 출력 결과
 
-### 3. 출력결과
+|필드|타입|설명|
+|---|---|---|
+|status|int|200 : 삭제 성공, 500 : 삭제 실패|
+|mainPageUrl|String|메인페이지 URL|
+
+## 팀 구해요 게시판 조회
+
+### API 기본정보
+
+|메서드|요청URL|출력포맷|설명|
+|---|---|---|---|
+|GET|/teams/borad/needteam|json|팀을 구하는 멤버들의 게시판 조회|
```

Comment:
The depth of the path should be meaningful and structured.

How about `board/findteam` or `board/needteam`?